packer/bsd/bsd.pkr.hcl

```diff
@@ -2,6 +2,15 @@ variable "bucket" { type = string }
 variable "gcp_project" { type = string }
```

Comment:
I think there's not really a need for the packer/bsd/ subdirectory anymore?

Reply:
Yes, moved them.